usagov/usagov-2021
# USAGOV-1336 W3C Markup Validation Errors

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1336

## Description

This pull request fixes a wide variety HTML validation issues revealed here:
https://validator.w3.org/nu/?doc=https%3A%2F%2Fwww.usa.gov%2Fbranches-of-government

Some commits focus on resolving the validator issues.
Other commits focus on code cleanup and reducing repetition.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions


* Visit localhost/branches-of-government
* View Page Source (Dont use dev tools to inspect since it automatically corrects some invalid HTML)
* Copy all the HTML
* Visit https://validator.w3.org/#validate_by_input
* Past the copied HTML into the textarea
* Click the `Check` button
* Confirm that there are no errors or warnings

### Change Requirements



### Validation Steps

- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
